prod-logic-data-index-postgresql-rhel8-image.yaml

```
@@ -0,0 +1,92 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+schema_version: 1
+
+name: "openshift-serverless-1/logic-data-index-postgresql-rhel8"
+version: "1.32.0"
+from: "registry.access.redhat.com/ubi8/ubi-minimal:latest"
```

```suggestion
from: "registry.access.redhat.com/ubi8/openjdk-17:latest"
```

Then you can remove our installation of Java. This module will be removed soon from upstream.

---

is that openjdk17 container productized?

---

As they are in `registry.access.redhat.com` it is fine to use it. I can do this change and use the jdk17 image and stop installing jdk 17.

---

@mareknovotny yes, they are and they are being used by Keycloak, Camel, Quarkus, and so on.